tests/unit/raw_send/kernel_func.hpp

```diff
@@ -34,7 +34,8 @@ struct raw_send_with_2_source_and_1_destination_func {
     offsets += offset;
     xetla_vector<uint64_t, SIMD> dsec = offsets + (uint64_t)a;
 
-    xetla_vector<dtype, SIMD> A_load_vec = xetla_load_global(a, offsets);
+    xetla_vector<dtype, SIMD> A_load_vec =
+        xetla_load_global<dtype, SIMD, 1>(a, offsets);
```

Comment:
why now template argument can't be inferred from function arguments ?

Reply:
The third template parameter VS = 1 or non-1 is 2 different specializations. VS is a new template parameter added this time

Reply:
Is there a third template parameter corresponding to 2 different template functions